plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -76,12 +76,30 @@ function cleanup_hadoop_docker_containers() {
     cleanup_docker_containers "${DOCKER_COMPOSE_LOCATION}"
 }
 
-function termination_handler(){
+function termination_handler() {
     set +e
     cleanup_docker_containers "$@"
     exit 130
 }
 
+# Check that all arguments are the names of non-empty variables.
+function require_env() {
+    ( # Use a subshell to avoid errexit and preserve xtrace
```

Comment:
You don't need to errexit, since you don't `set +e` / `set -e` here.
You don't have to, since variable access is now properly guarded with proper `test -v` check first.